internal/runtimehandlerhooks/high_performance_hooks.go

```diff
@@ -211,7 +236,7 @@ func setIRQLoadBalancing(c *oci.Container, enable bool, irqSmpAffinityFile strin
 		return err
 	}
 	currentIRQSMPSetting := strings.TrimSpace(string(content))
-	newIRQSMPSetting, newIRQBalanceSetting, err := UpdateIRQSmpAffinityMask(lspec.Resources.CPU.Cpus, currentIRQSMPSetting, enable)
+	newIRQSMPSetting, newIRQBalanceSetting, err := filterCPUs(lspec.Resources.CPU.Cpus, currentIRQSMPSetting, enable)
```

Comment:
Is filterCPUs descriptive enough?

Reply:
What about using something like computeCPUmask or so?

Reply:
sounds good, will change